### ISSUE TITLE
Ability to edit/delete GPT's responses

### ISSUE THREAD

**Issue by danjustice2** [cogentapps/chat-with-gpt#107]
In the OpenAI playground, you can edit/delete GPT's responses in the conversation and even "train" the model to respond a certain way by writing messages on GPT's behalf.

I'd really like to see some functionality like that here!

**Comment by cogentapps** [comment]
Cool idea!

**Comment by kevhardy** [comment]
Yep this a great way to get GPT to do exactly what you want by essentially writing examples of answers you need. I'd love this feature and it's one of the main reasons I used the playground instead of the normal chatgpt site.
